pkg/identityManager/client.go

```diff
@@ -73,3 +73,31 @@ func (certManager *certificateIdentityManager) GetConfig(remoteClusterID, namesp
 		},
 	}, nil
 }
+
+// GetRemoteTenantNamespace returns the tenant control namespace that
+// the remote cluster assigned to this peering.
+func (certManager *certificateIdentityManager) GetRemoteTenantNamespace(remoteClusterID, namespace string) (string, error) {
```

Comment:
```suggestion
func (certManager *certificateIdentityManager) GetRemoteTenantNamespaceName(remoteClusterID, localTenantNamespaceName string) (string, error) {
```

Just a proposal to remark better what the function is doing (IIUC)